exercise-1.js

```diff
@@ -444,74 +711,148 @@
         */
 
         //your code here
-
-
+        function printRole() {
+            for (i=0; i<devs.length; i++){
```

Comment:
I believe the original 'devs' array got mutated at some point.  When executed, this prints only those devs who have a tech stack.  Per the example, those without a tech stack were to be printed with "is not a developer".

Also, this should be solved using an Array method rather than a 'for' loop.